### PR TITLE
Creates a vpc and store the ids in the ssm parameter stoere

### PR DESCRIPTION
**Pull request by JOMACSIT**

### CODE REVIEW

**Review by aaronayitey — APPROVED** [review]
Great job, Johnson!

**Review by BobNukunu — COMMENTED** [review]
LGTM

**Comment by Kattafuah** [comment]
Smart job Johnson!

**Comment by Blaboni** [comment]
Impressive  Jay

**Comment by khingarthur** [comment]
Great work

**Review by FrancisNnanaOpara — COMMENTED** [review]
LGTM. Great job Johnson

**Review by DelaDoreen — COMMENTED** [review]
LGTM

**Comment by khingarthur** [comment]
Great work

**Review by analystden — APPROVED** [review]
Great!

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -35,13 +35,13 @@ jobs:
...
-      uses: jakejarvis/s3-sync-action@master
-      with:
-        args: --acl public-read
-      env:
-        SOURCE_DIR: './githubactions/tf.tfstate'
-        AWS_REGION: 'us-west-2'
-        AWS_S3_BUCKET: 'day5-state-bucket'
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    # - name: Sync S3
```
Comment (JOMACSIT) [inline .github/workflows/actions.yml:38]: Please take out the commented code to remove the noise

**Review by dcube12345 — APPROVED** [review]
Excellent  job Jay...

**Review by Kattafuah — APPROVED** [review]
Great job Johnson!

**Comment by kessiey** [comment]
Great Job

**Comment by Gina1010** [comment]
Great work done!

**Review by DelaDoreen — APPROVED** [review]
looks great

**Comment by sethkavi** [comment]
Great job!

**Comment by johnlokoacor** [comment]
Great work

**Pull request merged**